prometheus/archive/install.sls

```diff
@@ -82,11 +82,9 @@ prometheus-archive-install-{{ name }}-file-directory:
       - user: prometheus-config-user-install-{{ name }}-user-present
       - group: prometheus-config-user-install-{{ name }}-user-present
 
-            {%- if grains.kernel|lower == 'linux' and 'config_file' in p.pkg.component[name] %}
```

Comment:
the formula support BSD, MacOS, etc... https://github.com/saltstack-formulas/prometheus-formula/blob/master/prometheus/osfamilymap.yaml#L176

Reply:
You're right. I removed it because it made no sense to me to manage the `service file` depending on the `config_file` parameter.
Re-added it, stripping the 2nd condition. Unless there's a valid reason for it? I can re-add it.

Reply:
> Unless there's a valid reason for it? I can re-add it.

I don't know. It's used here https://github.com/saltstack-formulas/prometheus-formula/pull/44/files#diff-3614164752fd23cd5f64fa807102f9eaL109 but seems to have a default value coming from I-dont-know-where ()

Reply:
https://travis-ci.com/github/saltstack-formulas/prometheus-formula/jobs/377587315#L1384

Reply:
It seems it was used to build the `service` file as a parameter to the binary, but the condition was skipping the whole service file if it was not declared.

I [moved the condition](https://github.com/saltstack-formulas/prometheus-formula/blob/b3907a3d2cce09ddbf3c7b9875cfbb10ddb6088a/prometheus/archive/install.sls#L106-L110) to the place where it seems to make more sense